### PR TITLE
Fix to 3819 - Filtering issues by tags on main screen issues

### PR DESCRIPTION
**Pull request by rvillablanca**
This PR solves the issue #3819 where you can't filter issues by pressing on the tags.

Because there is a Default label set, there are many tags with same color and name like `bug`, but they have differents id so when you filter by pressing in one tag, you are filtering only for one repo.

Maybe this could be improved with another PR.

### CODE REVIEW

**Comment by techknowlogick** [comment]
Just commenting here with issue ID so GitHub links them: #3819

File: routers/user/home.go
Comment (jonasfranz) [inline routers/user/home.go]: Replace this with:
```go
opts.Labels = ctx.Query("labels")
```

**Review by jonasfranz — CHANGES_REQUESTED** [review]

File: routers/user/home.go
Comment (rvillablanca) [reply routers/user/home.go]: @JonasFranzDEV done!

**Comment by rvillablanca** [comment]
I don't understand why Drone generates errors when compiling. I have updated the branch but It seems Drone couldn't get updated sources.
Ref commit: https://github.com/go-gitea/gitea/pull/3824/commits/7243ed9b4e19aad4b2073ed11d8488434f603d53

@techknowlogick Can you help me ?

**Comment by techknowlogick** [comment]
@rvillablanca This is an issue with Github where sometimes it caches old information. Let me re-trigger drone and likely the updated information will be loaded.

**Comment by lafriks** [comment]
@JonasFranzDEV need your approval

**Comment by naiba** [comment]
@JonasFranzDEV @lunny @lafriks Hi, Can you merge this PR? can't wait for #5131

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
@naiba This needs two successful approvals before it can be merged.

**Comment by lunny** [comment]
@JonasFranzDEV need your approval.

**Review by jonasfranz — APPROVED** [review]

**Comment by techknowlogick** [comment]
@rvillablanca thanks for the PR. Would you be able to backport this commit to the release/v1.6 branch? If you need any assistance or clarification please let me know and I'd be happy to help.

**Comment by rvillablanca** [comment]
@techknowlogick I have created a PR #5219 for this purpose :+1:

**Comment by lunny** [comment]
@techknowlogick I think we should keep the backport label even if this is merged.

**Comment by techknowlogick** [comment]
@lunny ok.

**Pull request merged**
